src/FsAutoComplete/LspServers/AdaptiveFSharpLspServer.fs

```diff
@@ -416,13 +416,16 @@ type AdaptiveFSharpLspServer
   do
     disposables.Add
     <| fileChecked.Publish.Subscribe(fun (parseAndCheck, volatileFile, ct) ->
-      async {
-        let config = config |> AVal.force
-        do! builtInCompilerAnalyzers config volatileFile parseAndCheck
-        do! runAnalyzers config parseAndCheck volatileFile
+      if volatileFile.Source.Length = 0 then
```

Comment:
Probably should try to not expose indices like this.  Maybe use the `IsEmpty` suggestion above.

Reply:
What do you mean by indices? There is no index here. It's just a length test.